lib/element.js

```diff
@@ -305,12 +305,24 @@ class Element extends EventEmitter {
   }
 
   /**
+   * Determine if the key of the parent element is editable.
+   *
+   * @returns {Boolean} If the parent's key is editable.
+   */
+  isParentEditable() {
+    if (this.parent && 'currentKey' in this.parent) {
```

Comment:
This feels like a hack because sometimes this.parent is an Element type and sometimes its Document if it's the root-level element, so I can't just call this.parent.isKeyEditable because it will error on the root element.  Can't change that without bigger changes, so not sure the better way to check.